tests/test_bench.py

```
@@ -0,0 +1,23 @@
+import common
+
+
+def setup_module(module):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)